src/main/java/org/jabref/logic/importer/fetcher/CollectionOfComputerScienceBibliographiesFetcher.java

```diff
@@ -0,0 +1,36 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.Parser;
+import org.jabref.logic.importer.SearchBasedParserFetcher;
+import org.jabref.logic.importer.fetcher.CollectionOfComputerScienceBibliographiesParser;
+
+import com.microsoft.applicationinsights.core.dependencies.http.client.utils.URIBuilder;
```

Comment:
```suggestion
import org.apache.http.client.utils.URIBuilder;
```
Seems like you got the wrong import

Reply:
Thanks, I've fixed this now!